lib/introspector.js

```diff
@@ -41,7 +41,10 @@ const parseComments = (fn, text, start) => {
   return comments
     .map(line => line.trimRight())
     .filter(
-      line => line.length !== 0 && !line.startsWith('eslint-disable-next-line')
+      line =>
+        line.length !== 0 &&
+        !line.startsWith('eslint-disable-next-line') &&
```

Comment:
You should probably change this in a follow-up PR.
```suggestion
        !line.startsWith('eslint-disable') &&
```